wig/wig.go

```diff
@@ -139,3 +139,15 @@ func WriteToFileHandle(file io.Writer, rec Wig) {
 
 	}
 }
+
+// ChromToSlice returns the values from a wig entry matching a user-specified chromosome name
+func ChromToSlice(w []Wig, chrom string) []float64 {
+	var output []float64
+	for _, v := range w {
+		if v.Chrom == chrom {
+			output = make([]float64, len(v.Values))
+			output = v.Values
```

Comment:
Do we need each wig entry to be an entire chromosome?  I think this is the case for most of the wigs we make with gonomics, but the actual wig spec is much more flexible. Should we have a check in this loop to make sure we don't get more than one wig of the same chromosome and that it starts at zero with a step of 1?